File: gameserver/rpcgenout/gs.test.RPCManagerClient.java
// modified by ket.kio.RPCGen at Sat May 06 17:08:06 CST 2017.

package i3k.gs.test;

import ket.kio.SimplePacket;
import ket.kio.NetManager;
import i3k.rpc.Packet;
import i3k.SBean;

public class RPCManagerClient
{

	//// begin handlers.
	public void onTCPGameClientOpen(TCPGameClient peer)
	{
		// TODO
	}

	public void onTCPGameClientOpenFailed(TCPGameClient peer, ket.kio.ErrorCode errcode)
	{
		// TODO
	}

	public void onTCPGameClientClose(TCPGameClient peer, ket.kio.ErrorCode errcode)
	{
		// TODO
	}

	public void onTCPGameClientRecvServerChallenge(TCPGameClient peer, Packet.S2C.ServerChallenge packet)
	{
		// TODO
	}

	public void onTCPGameClientRecvServerResponse(TCPGameClient peer, Packet.S2C.ServerResponse packet)
	{
		// TODO
	}

	public void onTCPGameClientRecvLuaChannel(TCPGameClient peer, Packet.S2C.LuaChannel packet)
	{
		// TODO
	}

	public void onTCPGameClientRecvLuaChannel2(TCPGameClient peer, Packet.S2C.LuaChannel2 packet)
	{
		// TODO
	}

	public void onTCPGameClientRecvStrChannel(TCPGameClient peer, Packet.S2C.StrChannel packet)
	{
		// TODO
	}

	//// end handlers.
}

File: gameserver/rpcgenout/gs_test_rpcmanagerclient.cpp
// modified by ket.kio.RPCGen at Sat May 06 17:08:06 CST 2017.

#include "rpcmanagerclient.h"

namespace I3K
{

	//// begin handlers.
	void RPCManagerClient::OnTCPGameClientOpen(TCPGameClient* /*pPeer*/)
	{
		// TODO
	}

	void RPCManagerClient::OnTCPGameClientOpenFailed(TCPGameClient* /*pPeer*/, KET::KIO::ErrorCode /*errcode*/)
	{
		// TODO
	}

	void RPCManagerClient::OnTCPGameClientClose(TCPGameClient* /*pPeer*/, KET::KIO::ErrorCode /*errcode*/)
	{
		// TODO
	}

	void RPCManagerClient::OnTCPGameClientRecvServerChallenge(TCPGameClient* /*pPeer*/, const I3K::Packet::S2C::ServerChallenge * /*pPacket*/)
	{
		// TODO
	}

	void RPCManagerClient::OnTCPGameClientRecvServerResponse(TCPGameClient* /*pPeer*/, const I3K::Packet::S2C::ServerResponse * /*pPacket*/)
	{
		// TODO
	}

	void RPCManagerClient::OnTCPGameClientRecvLuaChannel(TCPGameClient* /*pPeer*/, const I3K::Packet::S2C::LuaChannel * /*pPacket*/)
	{
		// TODO
	}

	void RPCManagerClient::OnTCPGameClientRecvLuaChannel2(TCPGameClient* /*pPeer*/, const I3K::Packet::S2C::LuaChannel2 * /*pPacket*/)
	{
		// TODO
	}

	void RPCManagerClient::OnTCPGameClientRecvStrChannel(TCPGameClient* /*pPeer*/, const I3K::Packet::S2C::StrChannel * /*pPacket*/)
	{
		// TODO
	}

	//// end handlers.
}

